models/calendar/src/migration.ts

```diff
@@ -13,34 +13,70 @@
 // limitations under the License.
 //
 
-import core, { TxOperations } from '@hcengineering/core'
+import { Calendar, Event } from '@hcengineering/calendar'
+import core, { Ref, TxOperations } from '@hcengineering/core'
 import { MigrateOperation, MigrationClient, MigrationUpgradeClient } from '@hcengineering/model'
 import calendar from './plugin'
+import { DOMAIN_CALENDAR } from '.'
+import contact from '@hcengineering/contact'
 
-async function createSpace (tx: TxOperations): Promise<void> {
-  const current = await tx.findOne(core.class.Space, {
-    _id: calendar.space.PersonalEvents
+async function migrateCalendars (tx: TxOperations): Promise<void> {
+  const existCalendars = new Set((await tx.findAll(calendar.class.Calendar, {})).map((p) => p._id))
+  const users = await tx.findAll(contact.class.EmployeeAccount, {})
+  for (const user of users) {
+    if (!existCalendars.has(`${user._id}_calendar` as Ref<Calendar>)) {
+      await tx.createDoc(
+        calendar.class.Calendar,
+        core.space.Space,
+        {
+          name: user.email,
+          description: '',
+          archived: false,
+          private: false,
```

Comment:
Should it be private?

Reply:
No, we should have possibility to show events to another users in future